tests/schedules/test_clocks.py

```diff
@@ -445,6 +449,20 @@ def test_cron_clock_events_with_after_argument(self):
             start_date.add(days=3),
         ]
 
+    def test_cron_clock_events_with_day_or(self):
```

Comment:
It would be good to test with `day_or` as `None` and `True` as well - this would have caught the bug that was introduced above. You might use `pytest.mark.parametrize` to handle both cases with the same test code.

Reply:
Good catch! On it